aegis-web/yo/app/scripts/js/util.js

```diff
@@ -194,3 +194,11 @@ function skipTrailingSlash(path) {
 jQuery(document).on('click', '.mega-dropdown', function(e) {
   e.stopPropagation();
 });
+
+jQuery(document).on('click', '.dropdown', function(e) {
+    if($(this).hasClass('open')) {
+        $(this).removeClass('open');
+    } else {
+        $(this).addClass('open');
+    }
+});
```

Comment:
Why would you write this code when you could use Bootstrap's dropdown? 
Check your local style guide: http://localhost:9001/styleguide.html#dropdown

Reply:
removed and corrected